mkt/extensions/validation.py

```diff
@@ -0,0 +1,92 @@
+import json
+from zipfile import BadZipfile, ZipFile
+
+from rest_framework.exceptions import ParseError
```

Comment:
I think I would have preferred a custom `Exception` class that would be caught and re-thrown as `ParseError` in the view to make the validation code more agnostic, but it doesn't really matter now